command/cp.go

```diff
@@ -749,8 +750,12 @@ func prepareLocalDestination(
 	}
 
 	obj, err := client.Stat(ctx, dsturl)
-	if err != nil && err != storage.ErrGivenObjectNotFound {
-		return nil, err
+
+	if err != nil {
+		var givenObjNotFoundErr *storage.ErrGivenObjectNotFound
```

Comment:
Variable name is too long for me :) Let's rename them as `objNotFound`

Reply:
Fixed 👍

Reply:
Thanks.